src/fake-timers-src.js

```diff
@@ -267,6 +267,22 @@ function withGlobal(_global) {
             throw new Error("Callback must be provided to timer calls");
         }
 
+        if (isNearInfiniteLimit) {
+            timer.error = new Error();
+        }
+
```

Comment:
This was added in https://github.com/sinonjs/fake-timers/commit/52d6dba2ede53fd0be88ff750f59937fbb10daee, but this flag has been gone for some time. Is undefined.

Reply:
Oh, I see, it is a part of the reverted merge that is now gone? Should I rebase again?